source/manual/replicate-app-data-locally.html.md

```diff
@@ -36,54 +36,55 @@ want to do developer replication you will need to:
 
-If you don't see an ARN ID next to `Assigned MFA device` click the edit button and set one up.
+    If you don't see an ARN ID next to `Assigned MFA device` click the edit button and set one up.
 
 1. Create an access token
 
-Under `Access keys`click `Create access key` - you will secret is only displayed once, however if you fail to note it down just remove it and create another.
+    Under `Access keys`click `Create access key` - you will secret is only displayed once, however if you fail to note it down just remove it and create another.
 
 1. Install the AWS CLI application locally
 
-On MacOS this can be done using `brew install awscli`. This is being installed locally with the assumption you will be doing a two step backup process, i.e. download the files on your local machine and then update you VM. This is the recommended process as the download is quicker to you local machine for most users.
+    On MacOS this can be done using `brew install awscli`. This is being installed locally with the assumption you will be doing a two step backup process, i.e. download the files on your local machine and then update you VM. This is the recommended process as the download is quicker to you local machine for most users.
 
 1. Setup your AWS access config and credentials files
 
-I am repeating the instructions described in the [aws setup guide](/manual/user-management-in-aws.html) with one difference. In the credentials file the setting must be under default as otherwise they are not found by the replication script.
+    I am repeating the instructions described in the [aws setup guide](/manual/user-management-in-aws.html) with one difference. In the credentials file the setting must be under default as otherwise they are not found by the replication script.
 
-Create a `~/.aws/config` file:
+    Create a `~/.aws/config` file:
 
-```
-[profile govuk-integration]
-role_arn = <Role ARN>
-mfa_serial = <MFA ARN>
-source_profile = gds
-region = eu-west-1
-```
+    ```
+    [profile govuk-integration]
+    role_arn = <Role ARN>
+    mfa_serial = <MFA ARN>
+    source_profile = gds
+    region = eu-west-1
+    ```
 
-Role ARN: should be one of the following depending on the level of access you have been configured with [here](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-security/integration/common.tfvars):
+    Role ARN: should be one of the following depending on the level of access you have been configured with [here](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-security/integration/common.tfvars):
 
-* govuk-admins: arn:aws:iam::210287912431:role/govuk-administrators
-* govuk-poweruser: arn:aws:iam::210287912431:role/govuk-poweruser
-* govuk-users: arn:aws:iam::210287912431:role/govuk-users
+        * govuk-admins: arn:aws:iam::210287912431:role/govuk-administrators
+        * govuk-poweruser: arn:aws:iam::210287912431:role/govuk-poweruser
+        * govuk-users: arn:aws:iam::210287912431:role/govuk-users
```

Comment:
I don't think this bit needs to be indented this much, as I think this will lead to it being rendered as code, rather than a list.

Reply:
I thought that too, but without it the subsequent paragraph rendered as a code block. It looked fine when GitHub rendered the markdown but not when rendered in the tech docs template.